source/MRMesh/MRVoxelsVolume.h

```diff
@@ -40,7 +40,11 @@ struct VoxelsVolume
     Vector3i dims;
     Vector3f voxelSize{ 1.f, 1.f, 1.f };
 
-    [[nodiscard]] size_t heapBytes() const { return MR::heapBytes( data ); }
+    [[nodiscard]] size_t heapBytes() const
+    {
+        // `...IfSupported` because `T` can be a functor (`std::function`), then we can't compute this.
+        return MR::heapBytesIfSupported( data );
```

Comment:
mb just create overload for MR::heapBytes with std::function arg?

Reply:
Done.